timescaledb/how-to-guides/update-timescaledb/index.md

```diff
@@ -1,94 +1,101 @@
-# Updating TimescaleDB versions [](update)
-
-The instructions below allow you to update TimescaleDB within the same major release
-version (for example, from TimescaleDB 2.1 to 2.2, or from 1.7 to 1.7.4). If you need
-to upgrade between TimescaleDB 1.x and 2.x, see our [separate upgrade document][update-tsdb-2]
-for detailed instructions.
-
-TimescaleDB supports **in-place updates only**: you don't need to dump and
-restore your data, and versions are published with automated migration scripts
-that convert any internal state if necessary.
-
-<highlight type="warning">
-There is currently no automated way to downgrade to an earlier release of TimescaleDB without setting up
-a new instance of PostgreSQL with a previous release of TimescaleDB and then using `pg_restore`
-from a backup.
+# Update TimescaleDB versions
+Update your TimescaleDB version with an in-place update. By updating in-place,
+you don't need to dump and restore your data. Automated migration scripts handle
+conversion of internal states where needed.
+
```

Comment:
Do we need to mention that updates usually do not do complex operations and that only metadata is changed? This could make it necessary for the user to run VACUUM, for example. This might complicate issues a little too much though, so not sure about that. Might be better to write that in the upgrade instructions for that particular release.

Reply:
I think it would be good to add a note here, then direct readers to the Release Notes for more information.

Can you give me a bit more information on this? In what situations does the user need to run `VACUUM`? How would they know (even if the answer is just "read the release notes")? Do they need to do this even if they have auto-vacuum on?

Reply:
They probably do not need it if auto-vaccum is in place, but this was just an example. We had a case before where we did expensive work in an upgrade, which caused problems for a lot of users, so whatever needs to be done will be different for different releases.